pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -140,7 +141,7 @@ func getFloatingIP(client *gophercloud.ServiceClient, opts floatingips.ListOpts)
 		return true, nil
 	})
 	if err != nil {
-		if isNotFound(err) {
+		if cpoerrors.IsNotFound(err) {
```

Comment:
@jichenjc what about using providerError or cloudError, as the error originating from cloud or provider, Its just a suggestion.

Reply:
ok, maybe after merge? we need some framework for the err handling later on..

Reply:
@adisky Is the `providerError or cloudError` you mentioned for the package name or the Error class name? IMHO, i don't think it's a good candidate for class name since it's not as explicit as `IsNotFound`.

Reply:
@lingxiankong neither, I was just talking about the alias
https://github.com/kubernetes/cloud-provider-openstack/pull/493/files#diff-0f6df884fc7fa7f975c90fda39b3431cR51

Reply:
Then fine, we could change if most of us feel unhappy about that :-)